docs/rules/untrimmed-blocktranslate.md

```
@@ -0,0 +1,40 @@
+```html
+{% blocktranslate %}This string will have {{ value }} inside.{% endblocktranslate %}
+```
+
+Use instead:
+
+```html
+{% blocktranslate trimmed %}This string will have {{ value }} inside.{% endblocktranslate %}
+```
+
+## Fix safety
+This rule's fix is marked as safe: it inserts ` trimmed` immediately after
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)